src/common/bulk_load.thrift

```diff
@@ -206,4 +206,5 @@ struct query_bulk_load_response
     // detailed bulk load state for each replica
     6:list<map<dsn.rpc_address, partition_bulk_load_state>> bulk_load_states;
     7:optional string                                       hint_msg;
+    8:bool                                                  is_bulk_loading;
```

Comment:
If `app_status` = BULK_LOAD::INVALID, it means this app is not executing bulk load, I think this param is redundant.